### PR TITLE
Implement post_compile.sh custom script functionality

### PR DESCRIPTION
**Pull request by yrashk**
Hi guys,

What would it take to incorporate this functionality?

Happy New Year!

### CODE REVIEW

**Comment by ericmj** [comment]
I cannot find any other buildpacks that supports running custom scripts during compilation. I think it should be done with mix tasks instead. We should keep the exports and the tasks can run once by checking if the env vars are defined.

File: bin/compile [hunk 1]
```diff
@@ -15,6 +15,9 @@ build_path=$(cd $1 && pwd)
 cache_path=$(cd $2 && pwd)
 env_path=$(cd $3 && pwd)
 
+export BUILDPACK_BUILD_PATH=${build_path}
+export BUILDPACK_CACHE_PATH=${cache_path}
+export BUILDPACK_ENV_PATH=${env_path}
```
Comment (ericmj) [inline bin/compile]: We don't need to export `env_path`, all env vars in there are exported by the buildpack.

**Comment by yrashk** [comment]
I tried to do this with a mix task, it is such an incovenience.

**Comment by ericmj** [comment]
@yrashk Do you still need this?

**Comment by yrashk** [comment]
I use multipack for this now

**Pull request closed without merging**